layout/paypal.ejs

```diff
@@ -0,0 +1,42 @@
+<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="paypal">
+  <input type="hidden" name="cmd" value="_xclick">
+  <input type="hidden" name="business" value="accounts@redbrick.dcu.ie">
+  <input type="hidden" name="item_name" value="Membership">
+  <input type="hidden" name="item_number" value="Membership Payment">
+  <div id="paypalform" class="paypal">
+    <div class="paypal__selection">
+      <div class="paypal__selection__detials">
+        <input type="radio" name="amount" value="4.50">
+        Student
```

Comment:
This probably doesn't impact the style of if but I think this should be encapsulated in `<label>`